static/js/components/dashboard/courses/GradeDetailPopup_test.js

```diff
@@ -1,33 +1,41 @@
-// @flow
+// @flow weak
```

Comment:
this disables errors from flow for the file, I turned it off because I was getting a whole bunch of spurious 'possibly undefined value' errors from Flow for `course.proctorate_exam_grades`. I just tried out making `proctorate_exam_grades` a required property on `Course` and there don't seem to be any problems with that, so that will let me remove the `weak` keyword.